service/models.py

```diff
@@ -1,227 +1,219 @@
         db.session.commit()
 
     def serialize(self):
-        """ Serializes a Item into a dictionary """
+        """ Serialize an item object into a dictionary """
         return {
-            "customer_id": self.customer_id,
-            "item_id": self.item_id, 
+            "id": self.id,
+            "shopcart_id": self.shopcart_id,
             "name": self.name,
             "quantity": self.quantity,
             "price": self.price
         }
 
     def deserialize(self, data):
         """
-        Deserializes a Item from a dictionary
+        Deserialize an item dictionary to object
 
         Args:
             data (dict): A dictionary containing the resource data
         """
         try:
-            self.customer_id = data["customer_id"]
-            self.item_id = data["item_id"]
+            self.id = data["id"]
+            self.shopcart_id = data["shopcart_id"]
             self.name = data["name"]
             self.quantity = data["quantity"]
             self.price = data["price"]
         except KeyError as error:
             raise DataValidationError(
-                "Invalid Item: missing " + error.args[0]
+                f"Invalid {type(self).__name__}: missing {error.args[0]}"
             ) from error
         except TypeError as error:
             raise DataValidationError(
-                "Invalid Item: body of request contained bad or no data - "
-                "Error message: " + error
+                f"Invalid {type(self).__name__}: failed to deserialize request body\nError message: {error}"
             ) from error
         return self
 
     @classmethod
     def init_db(cls, app):
-        """ Initializes the database session """
-        logger.info("Initializing database")
+        """ Initialize the Item Table """
+        logger.info(f"Start initializing the {cls.__name__} Table")
         cls.app = app
-        # This is where we initialize SQLAlchemy from the Flask app
-        db.init_app(app)
+        db.init_app(app)  # init the Flask app for SQLAlchemy
         app.app_context().push()
         db.create_all()  # make our sqlalchemy tables
+        logger.info(f"Done initializing the {cls.__name__} Table")
 
     @classmethod
-    def all(cls):
-        """ Returns all of the Items in the database """
-        logger.info("Processing all Item")
+    def get_all(cls):
+        """ Get all items in DB table """
+        logger.info(f"Get all {cls.__name__}s")
         return cls.query.all()
-
-    @classmethod
-    def find(cls, id):
-        """ Finds a Item by it's ID """
-        logger.info("Processing lookup for id %s ...", id)
-        return cls.query.get(id)
-
-    @classmethod
-    def find_by_name(cls, name):
-        """Returns all Items with the given name
-
-        Args:
-            name (string): the name of the Item you want to match
-        """
-        logger.info("Processing name query for %s ...", name)
-        return cls.query.filter(cls.name == name)
```

Comment:
What about comment the two functions for now since we may use them later?

Reply:
sounds good to me!